### ISSUE TITLE
Edwards curves are not compatible with ECDH

### ISSUE THREAD

**Issue by Sarkoxed** [tlsfuzzer/python-ecdsa#348]
After trying to initialize ECDH with Ed25519 and compute a shared secret it returns the following error:

`AttributeError: 'PrivateKey' object has no attribute 'secret_multiplier'`

I guess it's called `__s` in `eddsa.PrivateKey` class.

**Comment by tomato42** [comment]
yes, because that's Ed25519, ECDH operation over Curve25519 is X25519, that's not implemented.

it's documented in the README.md:
```
Key generation, siging and verifying is also supported for Ed25519 and Ed448 curves.
```
